package.json

```diff
@@ -33,18 +33,17 @@
     "zone.js": "^0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.1002.3",
-    "@angular-devkit/build-ng-packagr": "~0.1002.3",
-    "@angular/cli": "~10.2.3",
-    "@angular/compiler-cli": "^10.2.5",
-    "@angular/language-service": "^10.2.5",
-    "@types/jasmine": "^3.7.7",
+    "@angular-devkit/build-angular": "~0.1102.16",
+    "@angular/cli": "~11.2.16",
+    "@angular/compiler-cli": "^11.2.14",
+    "@angular/language-service": "^11.2.14",
+    "@types/jasmine": "~3.6.0",
     "@types/jasminewd2": "~2.0.6",
     "@types/node": "^15.12.2",
-    "codelyzer": "^6.0.2",
+    "codelyzer": "^6.0.0",
     "coveralls": "^3.0.3",
-    "jasmine-core": "^3.7.1",
-    "jasmine-spec-reporter": "^7.0.0",
+    "jasmine-core": "~3.6.0",
```

Comment:
Why have you downgraded the jasmine packages? If not for any particular reason, please revert.

Reply:
Angular cli migration take care of this automatically

Reply:
Will be done after the angular 12 upgrade